lib/services/feature_flag_service.dart

```diff
@@ -83,6 +84,15 @@ class FeatureFlagService {
     }
   }
 
+  bool enableSearchFeature() {
+    String email = Configuration.instance.getEmail();
+    if (email.endsWith("@ente.io") || email == 'ashilkn99+ente77@gmail.com') {
```

Comment:
nit:
`return email != null && (endsWith("@ente.io") || kDebugMode)`

With `kDebugMode`, we can avoid hard coding personal email id.